### PR TITLE
Fix for dev js booting

### PR DESCRIPTION
**Pull request by rich-nguyen**
There are some intermittent issues with the dev js loading. This was caused by using the prefetch system in dev, which was supposed to align prod and dev behaviour. The best fix is to remove the prefetch system, because it was pre-emptively downloading some bootstraps in the order of a few milliseconds (9ms when I measured). To balance out any performance risk, the "optimised" `enhanced-vendor` js has been removed, which was a separate 2KB download.

I kept the user timing marking to the standard app, it's still useful.

@jamesgorrie

### CODE REVIEW

File: static/src/javascripts/boot.js [hunk 1]
```diff
@@ -12,20 +12,19 @@ Now we can safely require one of those modules.
...
 
-Bundles we need to run: commercial + (enhanced-vendor + enhanced)*
-* Only if we detect we should run enhance
+Bundles we need to run: commercial + enhanced
+
+Only if we detect we should run enhance.
  */
 
 define([
     'Promise',
```
Comment (sndrs) [inline static/src/javascripts/boot.js]: do these methods all really need to be wrapped in promises? there's not much going on now, feel a bit like overkill?

**Comment by sndrs** [comment]
👍 other than comment - nice one

File: static/src/javascripts/boot.js (hunk 1 above)
Comment (rich-nguyen) [inline static/src/javascripts/boot.js]: Yeah, the thought did cross my mind. If you agree it's worth doing, I'll open another pr for that?

Comment (sndrs) [inline static/src/javascripts/boot.js]: okedoke

File: static/src/javascripts/boot.js [hunk 2]
```diff
@@ -39,14 +38,7 @@ define([
...
-        return domReadyPromise.then(function () {
-            return promiseRequire(['bootstraps/standard/main'])
-                .then(function (boot) { boot(); });
-        });
-    };
-
-    var bootEnhanced = function () {
-        return promiseRequire(['bootstraps/enhanced/main'])
+        return promiseRequire(['bootstraps/standard/main'])
             .then(function (boot) { boot(); });
```
Comment (regiskuckaertz) [inline static/src/javascripts/boot.js]: `.then(boot)`

Comment (rich-nguyen) [inline static/src/javascripts/boot.js]: Yeah, still a few of those around in this file. We may un-promise this library soon

**Pull request merged**